### ISSUE TITLE
401 Error even on the 0.4.9 version released

### ISSUE THREAD

**Issue by uncdp** [techyowl/homebridge-august-smart-locks#12]
This is not a full exception just part of the header
10/2/2020, 2:38:47 PM [AugustLocks] [August] Error: GET failed with: 401
    at IncomingMessage.<anonymous> (/hoobs/node_modules/tiny-json-http/bundle.js:1:2186)
    at IncomingMessage.emit (events.js:322:22)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  raw: IncomingMessage {
    _readableState: ReadableState {
      objectMode: false,
      highWaterMark: 16384,
      buffer: BufferList { head: null, tail: null, length: 0 },
      length: 0,
      pipes: null,
      pipesCount: 0,
      flowing: true,
      ended: true,
      endEmitted: true,
      reading: false,
      sync: true,
      needReadable: false,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: false,
      emitClose: true,
      autoDestroy: false,
      destroyed: false,
      defaultEncoding: 'utf8',
      awaitDrain: 0,
      readingMore: true,
      decoder: null,
      encoding: null,
      [Symbol(kPaused)]: false
    },

**Comment by techyowl** [comment]
This is not enough information to investigate an issue off of. I need steps to reproduce. What steps you took that got you to this error. What configuration options you're using. What type of authentication method your august app uses (phone or email) and if your Homebridge version is up to date. I also need to know what platform you're running it on like a raspberry pi or computer Mac or linux or windows. More than likely you didn't run through the setup process correct.

1. uninstall the plugin from homebridge
2. remove any remaining configuration for the "AugustLocks" platform out of the json config.
3. Reinstall the Plugin.
4. configure the plugin with the default options setting a random string for the installation id and don't change this. Also, set the same authentication method you use for the official August App on your phone if it is a phone number only enter the phone number and if it is an email address only enter an email address. Be sure to not leave any leading or trailing spaces after anything. Then restart Homebridge and you should get an 2FA code to your login method such as your email address or phone number open the configuration back up and enter it in as a standard number no hyphens or anything other than the code itself. Then restart Homebridge one more time and after a few seconds it should find your locks that are available remotely in the August App. Bluetooth only locks will not show up. You must connect them via the August Connect hub one per lock or connect them directly for the wifi only models(not been tested)

**Comment by uncdp** [comment]
Hello, so had time today to run through these.
I am running homebrdige using Hoobs on Rasberry Pi 3B. All latest updates.

Steps as follows:
1) Uninstalled the plugin/cleared config, restarted (Physically).
2) Reinstalled the plugin, configured for phone auth started the plugin.
3) Received the code and added it, restarted. Received the code again (did nothing with it).
4) Checked logs still see same 401 error.
5) Entered previously mentioned new code, restarted. Again received the code again (did nothing with it).
6) Checked logs still see same 401 error.

Relevant Config looks as follows:
        {
            "platform": "AugustLocks",
            "plugin_map": {
                "plugin_name": "homebridge-august-smart-locks"
            },
            "name": "AugustLocks",
            "phone": "+xxxxxxxxxx",
            "password": "xxxxxxxxx",
            "installId": "i use random generated GUID, Should that be something else, how many characters are allowed",
            "code": "191231"
        }

**Comment by techyowl** [comment]
It's possible the guid is too many characters. I would suggest using a randomly generated alpha numeric string without any other symbols including the standard - found in UUIDv4  and my installation id is less than 64 characters

**Comment by uncdp** [comment]
Tried still same issue. Is it possible it has something to do with Hoobs it self. Do you know if any other hoobs folks use the plugin?
I tried a clean install and still get the same issue.
10/5/2020, 3:04:31 PM Check +1xxxxxxxxxx for your validation code
10/5/2020, 3:04:31 PM [AugustLocks] [August] getting locks ...
10/5/2020, 3:04:31 PM [AugustLocks] [August] Error: GET failed with: 401

/hoobs/node_modules/tiny-json-http/bundle.js - seems to though a parsing error.

**Comment by techyowl** [comment]
I haven't tested it with hooks recently so it may be incompatible with hoobs at the moment however it is functional on homebridge running from macOS.

**Comment by techyowl** [comment]
I can confirm it does not work with Hoobs which is what this issue is about. It likely is something to do with the recent platform changes. I will need to check it when I have some extra time. Sorry about the inconvenience, however you are welcome to setup homebridge to use this plugin for the time being.

**Comment by uncdp** [comment]
I appreciate the effort and will stand by. I will try a standalone homebridge install.

**Comment by bvk0118** [comment]
> This is not enough information to investigate an issue off of. I need steps to reproduce. What steps you took that got you to this error. What configuration options you're using. What type of authentication method your august app uses (phone or email) and if your Homebridge version is up to date. I also need to know what platform you're running it on like a raspberry pi or computer Mac or linux or windows. More than likely you didn't run through the setup process correct.
> 
> 1. uninstall the plugin from homebridge
> 2. remove any remaining configuration for the "AugustLocks" platform out of the json config.
> 3. Reinstall the Plugin.
> 4. configure the plugin with the default options setting a random string for the installation id and don't change this. Also, set the same authentication method you use for the official August App on your phone if it is a phone number only enter the phone number and if it is an email address only enter an email address. Be sure to not leave any leading or trailing spaces after anything. Then restart Homebridge and you should get an 2FA code to your login method such as your email address or phone number open the configuration back up and enter it in as a standard number no hyphens or anything other than the code itself. Then restart Homebridge one more time and after a few seconds it should find your locks that are available remotely in the August App. Bluetooth only locks will not show up. You must connect them via the August Connect hub one per lock or connect them directly for the wifi only models(not been tested)

I followed same exact steps but still doesn't work. Below is the entire exception I get in HomeBridge:

[AugustLocks] [August] Error: GET failed with: 401
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/homebridge-august-smart-locks/node_modules/tiny-json-http/bundle.js:1:2186)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1223:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  raw: IncomingMessage {
    _readableState: ReadableState {
      objectMode: false,
      highWaterMark: 16384,
      buffer: BufferList { head: null, tail: null, length: 0 },
      length: 0,
      pipes: null,
      pipesCount: 0,
      flowing: true,
      ended: true,
      endEmitted: true,
      reading: false,
      sync: true,
      needReadable: false,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: false,
      emitClose: true,
      autoDestroy: false,
      destroyed: false,
      defaultEncoding: 'utf8',
      awaitDrainWriters: null,
      multiAwaitDrain: false,
      readingMore: true,
      decoder: null,
      encoding: null,
      [Symbol(kPaused)]: false
    },
    readable: false,
    _events: [Object: null prototype] { end: [Array], data: [Function] },
    _eventsCount: 2,
    _maxListeners: undefined,
    socket: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'api-production.august.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'api-production.august.com',
      _readableState: [ReadableState],
      readable: true,
      _maxListeners: undefined,
      _writableState: [WritableState],
      writable: false,
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(asyncId)]: 184,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object]
    },
    connection: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'api-production.august.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'api-production.august.com',
      _readableState: [ReadableState],
      readable: true,
      _maxListeners: undefined,
      _writableState: [WritableState],
      writable: false,
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(asyncId)]: 184,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object]
    },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: {
      server: 'openresty/1.15.8.3',
      date: 'Thu, 08 Oct 2020 23:41:20 GMT',
      'content-type': 'application/json',
      'content-length': '65',
      connection: 'close',
      'api-version': '0.0.1',
      'x-response-time': '27'
    },
    rawHeaders: [
      'Server',
      'openresty/1.15.8.3',
      'Date',
      'Thu, 08 Oct 2020 23:41:20 GMT',
      'Content-Type',
      'application/json',
      'Content-Length',
      '65',
      'Connection',
      'close',
      'api-version',
      '0.0.1',
      'x-response-time',
      '27'
    ],
    trailers: {},
    rawTrailers: [],
    aborted: false,
    upgrade: false,
    url: '',
    method: null,
    statusCode: 401,
    statusMessage: 'Unauthorized',
    client: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'api-production.august.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'api-production.august.com',
      _readableState: [ReadableState],
      readable: true,
      _maxListeners: undefined,
      _writableState: [WritableState],
      writable: false,
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(asyncId)]: 184,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object]
    },
    _consuming: false,
    _dumped: false,
    req: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 2,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      socket: [TLSSocket],
      connection: [TLSSocket],
      _header: 'GET /users/locks/mine HTTP/1.1\r\n' +
        'x-august-api-key: 7cab4bbd-2693-4fc1-b99b-dec0fb20f9d4\r\n' +
        'x-kease-api-key: 7cab4bbd-2693-4fc1-b99b-dec0fb20f9d4\r\n' +
        'Content-Type: application/json\r\n' +
        'Accept-Version: 0.0.1\r\n' +
        'User-Agent: August/Luna-3.2.2\r\n' +
        'x-august-access-token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpbnN0YWxsSWQiOiIyODEyMjAxNyIsImFwcGxpY2F0aW9uSWQiOiIiLCJ1c2VySWQiOiI3MmUyOGI4YS0yOGM3LTQ3ZjktYjIxOC0xZTNjY2RiNTAzYWEiLCJ2SW5zdGFsbElkIjpmYWxzZSwidlBhc3N3b3JkIjp0cnVlLCJ2RW1haWwiOmZhbHNlLCJ2UGhvbmUiOmZhbHNlLCJoYXNJbnN0YWxsSWQiOnRydWUsImhhc1Bhc3N3b3JkIjp0cnVlLCJoYXNFbWFpbCI6dHJ1ZSwiaGFzUGhvbmUiOnRydWUsImlzTG9ja2VkT3V0IjpmYWxzZSwiY2FwdGNoYSI6IiIsImVtYWlsIjpbXSwicGhvbmUiOltdLCJleHBpcmVzQXQiOiIyMDIxLTAyLTA1VDIzOjQxOjIwLjAzNloiLCJ0ZW1wb3JhcnlBY2NvdW50Q3JlYXRpb25QYXNzd29yZExpbmsiOiIiLCJpYXQiOjE2MDIyMDA0ODAsImV4cCI6MTYxMjU2ODQ4MCwiTGFzdE5hbWUiOiJEaGFuYW5pIiwiRmlyc3ROYW1lIjoiSXNoYW50In0.cyGSHunE8KihszEaMNtRFjlLx6Xe3iMV4w4x09f0yH4\r\n' +
        'Content-Length: 0\r\n' +
        'Host: api-production.august.com\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: noopPendingOutput],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      insecureHTTPParser: undefined,
      path: '/users/locks/mine',
      _ended: true,
      res: [Circular],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'api-production.august.com',
      protocol: 'https:',
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    [Symbol(kCapture)]: false
  },
  body: '[object Object]',
  statusCode: 401

**Comment by carson-autocicd** [comment]
Following the same steps as the first commenter, I get the same error. One thing that's interesting is every time it restarts, I get a new 2FA code from August. 

`[10/25/2020, 12:06:39 PM] [AugustLocks] [August] getting locks ...
[10/25/2020, 12:06:39 PM] [AugustLocks] [August] Error: GET failed with: 401
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-august-smart-locks/node_modules/tiny-json-http/bundle.js:1:2186)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (_stream_readable.js:1220:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  raw: IncomingMessage {
    _readableState: ReadableState {
      objectMode: false,
      highWaterMark: 16384,
      buffer: BufferList { head: null, tail: null, length: 0 },
      length: 0,
      pipes: null,
      pipesCount: 0,
      flowing: true,
      ended: true,
      endEmitted: true,
      reading: false,
      sync: true,
      needReadable: false,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: false,
      emitClose: true,
      autoDestroy: false,
      destroyed: false,
      defaultEncoding: 'utf8',
      awaitDrainWriters: null,
      multiAwaitDrain: false,
      readingMore: true,
      decoder: null,
      encoding: null,
      [Symbol(kPaused)]: false
    },
    readable: false,
    _events: [Object: null prototype] { end: [Array], data: [Function] },
    _eventsCount: 2,
    _maxListeners: undefined,
    socket: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'api-production.august.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'api-production.august.com',
      _readableState: [ReadableState],
      readable: true,
      _maxListeners: undefined,
      _writableState: [WritableState],
      writable: false,
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(asyncId)]: 235,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object]
    },
    connection: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'api-production.august.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'api-production.august.com',
      _readableState: [ReadableState],
      readable: true,
      _maxListeners: undefined,
      _writableState: [WritableState],
      writable: false,
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(asyncId)]: 235,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object]
    },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: {
      server: 'openresty/1.15.8.3',
      date: 'Sun, 25 Oct 2020 19:06:39 GMT',
      'content-type': 'application/json',
      'content-length': '65',
      connection: 'close',
      'api-version': '0.0.1',
      'x-response-time': '2'
    },
    rawHeaders: [
      'Server',
      'openresty/1.15.8.3',
      'Date',
      'Sun, 25 Oct 2020 19:06:39 GMT',
      'Content-Type',
      'application/json',
      'Content-Length',
      '65',
      'Connection',
      'close',
      'api-version',
      '0.0.1',
      'x-response-time',
      '2'
    ],
    trailers: {},
    rawTrailers: [],
    aborted: false,
    upgrade: false,
    url: '',
    method: null,
    statusCode: 401,
    statusMessage: 'Unauthorized',
    client: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'api-production.august.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'api-production.august.com',
      _readableState: [ReadableState],
      readable: true,
      _maxListeners: undefined,
      _writableState: [WritableState],
      writable: false,
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(asyncId)]: 235,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object]
    },
    _consuming: false,
    _dumped: false,
    req: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 2,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      socket: [TLSSocket],
      connection: [TLSSocket],
      _header: 'GET /users/locks/mine HTTP/1.1\r\n' +
        'x-august-api-key: removed
        'x-kease-api-key: removed
        'Content-Type: application/json\r\n' +
        'Accept-Version: 0.0.1\r\n' +
        'User-Agent: August/Luna-3.2.2\r\n' +
        'x-august-access-token: Removed
        'Content-Length: 0\r\n' +
        'Host: api-production.august.com\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _onPendingData: [Function: noopPendingOutput],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      insecureHTTPParser: undefined,
      path: '/users/locks/mine',
      _ended: true,
      res: [Circular],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    [Symbol(kCapture)]: false
  },
  body: '[object Object]',
  statusCode: 401`

**Comment by techyowl** [comment]
This has been resolved in 5.0.1 or greater. Feel free to update when you have a chance. I also confirmed compatibility with this version for Hoobs. Upgrade steps are as follows:

1. update the version when it is available
2. remove any 2FA code and restart homebridge/hoobs
3. add the code you receive to the configuration for the plugin.
4. restart homebridge/hoobs one last time
5. august smart locks should now be available in the accessories list.
